EugeneFoodScene/Client/wwwroot/css/app.css

```diff
@@ -505,5 +505,101 @@ app {
         margin-left: 0;
     }
 }
+
+/* header */
+h1, h2, h3 {
```

Comment:
try using a class instead of element style.  we don't want to override all the headers without knowing if they look OK.